cmd/init.go

```diff
@@ -77,39 +77,55 @@ var initCmd = &cobra.Command{
 
 		return c.Run()
 	},
-	Long: `Initialize a ksonnet project in a new directory,` + " `app-name`" + `. This process
-consists of two steps:
-
-1. Generating ksonnet-lib. Users can set flags to generate the library based on
-   a variety of data, including server configuration and an OpenAPI
-   specification of a Kubernetes build. By default, this is generated from the
-   capabilities of the cluster specified in the cluster of the current context
-   specified in` + " `$KUBECONFIG`" + `.
-2. Generating the following tree in the current directory.
-
-        app-name/
-          .gitignore     Default .gitignore; can customize VCS
-          .ksonnet/      Metadata for ksonnet
-          environments/
-            default/      [Default generated environment]
-          components/    Top-level Kubernetes objects defining application
-          lib/           user-written .libsonnet files
-          vendor/        part libraries, prototypes
+	Long: `Initialize a ksonnet application in a new directory,` + " `app-name`" + `.
+
+This command will generate all the project scafolding required to begin creating and
+deploying components to Kubernetes clusters.
+
+ksonnet applications are initialized based on your current cluster configurations,
+as defined in your` + " `$KUBECONFIG` " + `environment variable. See examples
+of how to customize configurations in the *Examples* section below.
+
+Creating a ksonnet application will result in the following directory tree.
+
+    app-name/
+      .ksonnet/      Metadata for ksonnet
+      app.yaml       Application specifications, ex: name, api version
+      components/    Top-level Kubernetes objects defining the application
+      environments/  Kubernetes cluster definitions
+        default/     Default environment, initialized from the current kubeconfig
+          .metadata/ [1]
```

Comment:
I like the footnote idea, but it might need some more explanation---consider something along the lines of the following:

```
Contains a versioned ksonnet-lib, see [1] for details
```